test/active_model_serializers/register_jsonapi_renderer_test_isolated.rb

```diff
@@ -111,18 +101,20 @@ def test_jsonapi_parser_registered
     end
 
     def test_jsonapi_renderer_registered
-      expected = {
-        'data' => {
-          'attributes' => {
-            'name' => 'Johnny Rico'
-          },
-          'type' => 'users'
-        }
-      }
+      expected = {'name' => 'Johnny Rico'}
+
       payload = '{"data": {"attributes": {"name": "Johnny Rico"}, "type": "authors"}}'
       headers = { 'CONTENT_TYPE' => 'application/vnd.api+json' }
       post '/render_with_jsonapi_renderer', params: payload, headers: headers
-      assert expected, response.body
+      assert_equal expected, JSON.parse(response.body)["data"]["attributes"] 
+    end
+
+    def test_jsonapi_renderer_registered_withfields
+      expected = {'title'  => 'This is a test' }
+      payload = '{"data": {"attributes": {"title": "This is a test","body": "Of an emergency alert system."}, "type": "posts"},"fields": { "posts" : ["title"] } }'
+      headers = { 'CONTENT_TYPE' => 'application/vnd.api+json' }
+      post '/render_with_jsonapi_fields?fields[posts][]=title', params: payload, headers: headers
+      assert_equal expected, JSON.parse(response.body)["data"]["attributes"]
```

Comment:
I don't think this test belongs here.  (typo in name as well).  This test appears to be testing usage of the jsonapi renderer, where all this file is doing is testing registering it, which is why it's isolated